src/classes/BGE_DataImportBatchEntry_CTRL_TEST.cls

```diff
@@ -82,11 +82,15 @@ private class BGE_DataImportBatchEntry_CTRL_TEST {
     @isTest
     static void testGetNewDataImport() {
 
+        Boolean errorIsThrownForNullId = false;
+
         try {
             String jsonModel = BGE_DataImportBatchEntry_CTRL.getDataImportModel(null);
         } catch (AuraHandledException e) {
-            System.assert(e.getMessage().contains('Data Import Model must receive a Batch ID.'));
+            errorIsThrownForNullId = true;
```

Comment:
I think I fixed this elsewhere on a branch I thought would go in first, so we'll probably see merge conflicts from this as a heads up.